src/Microsoft.DotNet.Interactive.ApiCompatibility.Tests/ApiCompatibilityTests.Formatting_api_is_not_changed.approved.txt

```diff
@@ -292,6 +293,10 @@ Microsoft.DotNet.Interactive.Formatting.TabularData
     public System.Void Add(TableSchemaFieldDescriptor descriptor)
     public System.Boolean Contains(System.String name)
     public System.Collections.Generic.IEnumerator<TableSchemaFieldDescriptor> GetEnumerator()
+  public class TableDataFieldDescriptorsJsonConverter : Microsoft.DotNet.Interactive.Formatting.JsonConverter<TableDataFieldDescriptors>
```

Comment:
Does this need to be public?

Reply:
Suppose that as they are used as attributes they don't need to be